static/docs/get-started/data-artifacts.md

```diff
@@ -0,0 +1,45 @@
+# Reusing Data Artifacts
+
+We've explained how to [upload](/doc/get-started/share-data) and
+[download](/doc/get-started/retrieve-data) <abbr>data artifacts</abbr> (with
+`dvc pull` and `dvc push`) as a way to share them on remote storage. This is
+great for several people to work on the same <abbr>DVC project</abbr> (on
+different machines). But what if we wanted to reuse a dataset from a project in
+a completely different one?
+
+That's why we created `dvc import`! This commands can download data from any DVC
+project hosted on a Git repository. It also tracks the imported data by creating
+an import stage ([DVC-file](/doc/user-guide/dvc-file-format)). Example:
+
+```dvc
+$ dvc import https://github.com/iterative/dataset-registry \
+           get-started/data.xml
+```
+
+This downloads `data.xml` (adding it to `.gitignore`) and creates the
```

Comment:
Not sure how does it fit in the Get Started "story" .. this command will create a DVC-file that won't be used anywhere, right?

Reply:
Good point, didn't think about that... How about just adding a note like at the bottom of https://dvc.org/doc/get-started/configure? Suggested change in 1b3bdfa

Reply:
May be we can say something like - "An alternative way to initialize the project would be to use `dvc import` ... " and actually show how `add` can be replaced with import and what are the benefits?

Reply:
Yes, great idea. Improved in cb42034.